apps/app/src/components/QuestionsHeader/useQuestionsHeader.tsx

```diff
@@ -0,0 +1,22 @@
+import { usePathname, useRouter } from "next/navigation";
+import { ChangeEvent, useState } from "react";
+import { DEFAULT_ORDER_QUERY } from "../../lib/order";
+
+export const useQuestionsHeader = () => {
```

Comment:
Czy to na pewno dobra nazwa na hooka, który pozwala na sortowanie pytań? Raczej nie.
Może `useQuestionsSortBy`, który zwraca `{orderBy, order i setOrderBy}` ?